### PR TITLE
Convert tests to pipeline api

### PR DESCRIPTION
**Pull request by gurbuzali**

### CODE REVIEW

File: kafka-test/src/test/java/com/hazelcast/jet/tests/kafka/KafkaTest.java [hunk 1]
```diff
@@ -178,6 +128,27 @@ public void tearDown() {
...
+    private Pipeline pipeline() {
+        Pipeline pipeline = Pipeline.create();
+
+        pipeline.drawFrom(KafkaSources.<String, Trade>kafka(kafkaProps, topic))
+                .map(Map.Entry::getValue)
+                .addTimestamps(Trade::getTime, lagMs)
+                .window(sliding(windowSize, slideBy))
+                .groupingKey(Trade::getTicker)
+                .aggregate(counting())
+                .map(e -> {
```
Comment (cangencer) [inline kafka-test/src/test/java/com/hazelcast/jet/tests/kafka/KafkaTest.java]: you can use the overload of `aggregate` here and skip this mapping step

File: kafka-long-running-session-window-test/src/test/java/com/hazelcast/jet/tests/kafka/LongRunningKafkaSessionWindowTest.java [hunk 2]
```diff
@@ -134,48 +125,33 @@ public void kafkaTest() throws InterruptedException {
...
-                streamKafkaP(kafkaPropertiesForResults(brokerUri, offsetReset), noWatermarks(), topic + "-result"));
+    private Pipeline verificationPipeline() {
+        Pipeline pipeline = Pipeline.create();
 
-        long countCopy = countPerTicker;
-        Vertex verifyRecords = dag.newVertex("verification", () -> new VerificationSink(countCopy));
+        Properties properties = kafkaPropertiesForResults(brokerUri, offsetReset);
+        final int countCopy = countPerTicker;
+        pipeline.drawFrom(KafkaSources.kafka(properties, topic + "-results"))
+                .drainTo(Sinks.fromProcessor("verification",
```
Comment (cangencer) [inline kafka-long-running-session-window-test/src/test/java/com/hazelcast/jet/tests/kafka/LongRunningKafkaSessionWindowTest.java]: I think you can use the custom sink builder here too

**Comment by eminn** [comment]
Pipeline API🥇

File: kafka-long-running-session-window-test/src/test/java/com/hazelcast/jet/tests/kafka/LongRunningKafkaSessionWindowTest.java (hunk 2 above)
Comment (gurbuzali) [reply kafka-long-running-session-window-test/src/test/java/com/hazelcast/jet/tests/kafka/LongRunningKafkaSessionWindowTest.java]: 👍

File: kafka-test/src/test/java/com/hazelcast/jet/tests/kafka/KafkaTest.java (hunk 1 above)
Comment (gurbuzali) [reply kafka-test/src/test/java/com/hazelcast/jet/tests/kafka/KafkaTest.java]: 👍

**Pull request merged**